js/mainPage.js

```diff
@@ -0,0 +1,226 @@
+
+function loadSlider(){
+	//create the slider
+	var slider = document.getElementById('mySlider');
+	noUiSlider.create(slider, {
+    	start: [0, 100],
+    	connect: true,
+    	range: {
+        	'min': 0,
+        	'max': 100
+	    }
+	});
+	//event for the display of the slider range
+	slider.noUiSlider.on('update',function(values,handler){
+	document.getElementById("first").innerHTML="$"+values[0];
+	document.getElementById("second").innerHTML="$"+values[1];
+	$('.menu').innerHTML = "";	
+	loadElements(parseInt(values[0]),parseInt(values[1]));
+});
+	var ok=localStorage.getItem("itemsLength");
+	//check if there is data in lcoal storage
+	if(localStorage.getItem("itemsLength") === null)
+		writeElements();
+	loadElements(0,100);
+}
+//sets slider values to 0 a and 100
+function setSlider(){
+	var slider = document.getElementById('mySlider');
+	slider.noUiSlider.set([0, 100]);
+}
+//display the div with add/remove operations
+$(document).on('click','.item',function(){
+		$(".delAdd").fadeIn().delay(7000).fadeOut();
+		var clas = $(this).attr('class');
+		nr = clas.slice(5);
+		$(".del").click(function(){ deleteEl(nr); });
+	});
+
+//takes input and adds a new element
+function addEl(){
+	var title = prompt("Enter the name of the dish: ","Gourmet Croissant");
```

Comment:
it's probably better to leave a space after the `,` just like you would do in a sentence :D